src/mdesktopaction.h

```diff
@@ -0,0 +1,44 @@
+/***************************************************************************
+** This file was derived from the MDesktopAction implementation in the
+** libmeegotouch library.
+**
+** Copyright (C) 2021 Chupligin Sergey (NeoChapay) <neochapay@gmail.com>
+**
+** This library is free software; you can redistribute it and/or
+** modify it under the terms of the GNU Lesser General Public
+** License version 2.1 as published by the Free Software Foundation
+** and appearing in the file LICENSE.LGPL included in the packaging
+** of this file.
+**
+****************************************************************************/
+
+#ifndef MDESKTOPACTION_H
+#define MDESKTOPACTION_H
+
+#include <mlite-global.h>
+#include <QString>
+
+class MDesktopEntryPrivate;
+
+class MLITESHARED_EXPORT MDesktopAction
+{
+
+public:
+    MDesktopAction(const QString &fileName, const QString action);
+    QString name() const;
+    QString nameUnlocalized() const;
+    QString exec() const;
```

Comment:
Consistent with MDesktopEntry but exec() on action could be mistaken to execute something. Not sure on the options though.

Reply:
option in standart https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#extra-actions

Reply:
Yes I know where it's coming from but just saying directly mapping as c++ method creates ambiguity on the expectation. Compare to getExec() is clearly a getter and not expected to have side-effects. Not saying that would be a good name here, though.